App/style.css

```diff
@@ -146,7 +146,7 @@ body > .bar {
   pointer-events: none;
 }
 
-.envelope[hide=true],.text[hide=true],.label[hide=true],.button[hide=true] {
+*[hide=true] {
```

Comment:
@polytroper This was probably the riskiest change I made here, as I wanted a div to respect the 'hide' flag, and figured at the same time I could just make all elements do so...

Reply:
Yeah seems fine to me